### PR TITLE
Activity feed screen UI

### PR DESCRIPTION
**Pull request by saad-abid-crowdbotics**
## Activity feed screen

- [ ] Bugfix
- [x] New feature
- [ ] Minor changes

## Changes introduced
Activity feed screen

## Test and review
1. Activity feed screen user interface

image

### CODE REVIEW

File: modules/screen-activity-feed/index.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
+import * as React from "react";
+import { Text, View, Image, ScrollView, StyleSheet, TouchableHighlight } from "react-native";
+
+const pressed = () => {
+  console.log("pressed");
+};
+
+export const ActivityFeed = () => {
```
Comment (danielsousaio) [inline modules/screen-activity-feed/index.js:8]: Please change this to be a default export:
https://crowdbotics.github.io/modules/authoring#screen-modules

**Review by danielsousaio — CHANGES_REQUESTED** [review]
There's already a module called `screen-activity-feed`, please change the name of this one or make this an update to that module.

File: modules/screen-activity-feed/index.js (hunk 1 above)
Comment (saad-abid-crowdbotics) [reply modules/screen-activity-feed/index.js]: Resolved.

**Review by danielsousaio — CHANGES_REQUESTED** [review]
Thanks for submitting the PR.

Screen-only module review checklist:
- [x] `meta.json` root property starts with `/screens` path
- [ ] no dependencies on the `package.json` file
- [x] module name starts with `@screens`
- [x] includes `preview.png` file
- [x] module's `index.js` contains `export default`
- [x] module's directory name starts with `screen-`

Issues found:
- Missing a `package.json` file.

**Pull request merged**